src/services/export.js

```diff
@@ -291,18 +296,25 @@ const saveEpub = async (page, interactive) => {
   try {
     // todo: parse title in appropriate language
     introduction.title = 'Introduction';
-    introduction.data = await page.$eval('.description p', el => el.innerHTML);
+    introduction.data = await page.$eval('.description', el => el.outerHTML);
   } catch (err) {
     console.error(err);
   }
 
   // get body for epub
   // use the export class to differentiate from tools content
   const body = await page.$$eval('.export > section', phases =>
-    phases.map(phase => ({
-      title: phase.getElementsByClassName('name')[0].innerHTML,
-      data: phase.getElementsByClassName('resources')[0].innerHTML,
-    }))
+    phases.map(phase => {
+      const content = phase.getElementsByClassName('resources')[0].innerHTML;
```

Comment:
We should factor out the strings `resources` and `description` to the selectors file.

Reply:
I refactor them in the other pull request.